bigtable/client/table_admin.h

```diff
@@ -58,6 +58,23 @@ class TableAdmin {
       : impl_(std::move(client), std::move(instance_id),
               std::move(retry_policy), std::move(backoff_policy)) {}
 
+  /**
```

Comment:
The default implementation should work for these, please remove or replace with:

```C++
TableAdmin(TableAdmin const& rhs) = default;
TableAdmin& operator=(TableAdmin const& rhs) = default;
```

Reply:
You can remove the comments, it is pretty obvious what is going on.